cat-boot-report-pdf/src/main/java/cc/catalysts/boot/report/pdf/utils/ReportStaticElementOnPages.java

```diff
@@ -0,0 +1,12 @@
+package cc.catalysts.boot.report.pdf.utils;
+
+/**
+ * Created by sfarcas on 7/21/2016.
+ *
+ * position of the footer or the header:
+ * on all pages, on all pages except the first, on all pages except the last
+ *
+ */
+public enum ReportStaticElementOnPages {
```

Comment:
why not:

enum PositionOfStaticElements {
    OnAllPages,
    OnAllPagesButFirst,
    OnAllPagesButLast,
}